documentation/modules/ROOT/pages/configuration/applying-transformations-selectively.adoc

```diff
@@ -0,0 +1,190 @@
++
+For example:
++
+[source]
+----
+predicates.IsOutboxTable.pattern=outbox.event.*
+----
+
+. If you want to negate a condition, append the `negate` keyword to the transform alias and set it to `true`.
+
++
+For example:
++
+[source,options="nowrap" subs="+quotes"]
+----
+transforms.outbox.negate=true
+----
++
+The preceding property inverts the set of records that the predicate matches, so that Kafka Connect applies the transform to any record that does not match the condition specified in the predicate.
+
+.Example: TopicNameMatch predicate for the outbox event router transformation
+
+The following example shows a {prodname} connector configuration that applies the outbox event router transformation only to messages that Debezium emits to the Kafka `outbox.event.order` topic.
+
+Because the `TopicNameMatch` predicate evaluates to _true_ only for messages from the outbox table (`outbox.event.*`), the transformation is not applied to messages that originate from other tables in the database.
+
+[source]
+----
+transforms=outbox
+transforms.outbox.predicate=IsOutboxTable
+transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
+predicates=IsOutboxTable
+predicates.IsOutboxTable.type=org.apache.kafka.connect.predicates.TopicNameMatch
+predicates.IsOutboxTable.pattern=outbox.event.*
+
+----
+
+// Type: procedure
+[id="ignoring-tombstone-events"]
+== Ignoring tombstone events
+
+You can control whether {prodname} emits tombstone events, and how long Kafka retains them.
+Depending on your data pipeline, you might want to configure {prodname} so that it does not emit tombstone events.
+
+Whether you enable {prodname} to emit tombstones depends on how topics are consumed in your environment and by the characteristics of the sink consumer.
+Some sink connectors rely on tombstone events to remove records from downstream data stores.
+In cases where sink connectors rely on tombstone records to indicate when to delete records in downstream data stores, configure {prodname} to emit them.
+
+To ensure that sink connectors receive tombstone events, set the retention policy for topics, so that the connector are able to read the event messages before Kafka removes them during log compaction.
+The length of time that a topic retains tombstones before compaction is controlled by the link:{link-kafka-docs}#topicconfigs_delete.retention.ms[`delete.retention.ms`] property for the topic.
+
+Some sink connectors or downstream Kafka consumers are unable to process records with a null value.
+To avoid errors in sink connectors that cannot handle tombstone records, you can configure {prodname} to strip out tombstone records before it saves event records to Kafka topic.
+
+If you support downstream consumers that cannot process tombstones, rather than using the `RecordIsTombstone` predicate to remove tombstone messages after {prodname} emits them, you can prevent {prodname} from emitting tombstones in the first place.
```

Comment:
See my related comment above: I think it should be framed in a way that you can configure things so that tombstones _are_ emitted (allowing log compaction to remove all records of the affected key) but still can skip them on the sink side for connectors which don't expect them.